service/staging/staging_controller.py

```diff
@@ -40,6 +40,28 @@ def list_staging():
     tree = _list_dir(staging_dir)
     return jsonify(tree)
 
+@staging_controller.route(
+    '/<path:path>',
+    methods=['GET'],
+    strict_slashes=False
+)
+def get_path(path):
+    """
+    Forwards a file form staging controller
+
+    Returns HTTP status code 400 if file was not found
+
+    :param path:
+    :return:
```

Comment:
Hier noch dokumentieren, in welchem Fall JSON und in welchem Dateiinhalte zurückgegeben werden.